Studio-Lovelies/GG-JointJustice-Unity
# Added WebGL build to GitHub Actions

## Summary
This change **does not change game code**.

This PR adds **WebGL builds** to the GitHub Actions which are automatically run.

Note: WebGL builds **usually take around 2x** the amount of time of native builds, as Unity first builds a native build to then convert it to WebAssembly. **The time until test results are visible on GitHub does not change**, the "failed workflow" email notification is only sent **once all jobs are completed** however.

## Before/after screenshots and/or animated gif
![image](https://user-images.githubusercontent.com/1689033/159178293-2d6af028-2455-4aee-9c69-7d96a73b8f81.png)

## Testing instructions
1. Download the `WebGL-*` artifact of [any successful run of this branch](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/actions/runs/2012409460)
2. **IMPORTANT:** Open this folder via HTTP by either:
    1. uploading this folder to your own webhost
    2. use [this temporary version hosted on my personal webspace](https://vimaster.de/dump/Game%20Grumps%20-%20Joint%20Justice/)
3. Give it around a minute to load (this heavily varies based on download and processing speeds)
4. Verify the game runs as expected

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - closes #172
